roles/apache2/vars/main.yaml

```
@@ -86,12 +72,7 @@ _a2_install_ferm_svc: |-
 
 _a2_ferm_allow_from: |-
   {{
-    None
-    if not apache2__ferm_allow_from else
-    [apache2__ferm_allow_from]
-    if apache2__ferm_allow_from is not iterable or
-    apache2__ferm_allow_from is string
-    else apache2__ferm_allow_from
+    (apache2__ferm_allow_from | d([]) | tina_pm.common.to_list) or None
```

Same problem here, although `roles/apache2/templates/ferm.conf.j2` does not differentiate between None and an empty list so the `or None` is not needed:
```suggestion
    apache2__ferm_allow_from | d([], True) | tina_pm.common.to_list
```
or
```suggestion
    (apache2__ferm_allow_from or []) | tina_pm.common.to_list
```